src/js/utils/api.js

```diff
@@ -42,11 +51,13 @@ export default class api {
   static post(requestPath, payload) {
     return new Promise((resolve, reject) => {
       try {
-        if (requestPath !== 'notes') {
-          throw new Error('api.post() only \`notes\`');
+        if (requestPath !== NOTES_API_PATH) {
+          throw new Error(`api.post() only \`${NOTES_API_PATH}\``);
         }
         let data = JSON.parse(localStorage.getItem(NAMESPACE) || STRINGIFY_INITIAL_STORE);
-        data[requestPath].unshift(payload);
+        const tableName = apiPathToTableName(requestPath);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).